tests/drivers/st77xx/main.c

```diff
@@ -31,6 +31,94 @@
 #include "st77xx.h"
 #include "st77xx_params.h"
 
+#if MODULE_LCD_INIT_SEQUENCE
+
+#if MODULE_ST7735
+
+#include "st7735_internal.h"
+
+static const uint8_t st7735_default_init[] =  {
```

Comment:
I think those should live in the driver, not copy & pasted between applications/boards.
Or is there anything application specific about this?

(Sure it makes sense being able to supply an out-of-tree command sequence, but if we already have some known good init sequences for some boards, they should be easily usable)

Reply:
These init sequences are actually only intended for test and demonstration purposes. They have the disadvantage that they use the parameter macros directly and cannot be parameterized by the parameter set of the device. Furthermore, most of the commands in these example sequences only configure the default values after resetting the display anyway and are actually not needed at all. Last, the default init sequences are already implemented by the driver for the ST77xx variants.